src/scss/settings/_breakpoints.scss

```diff
@@ -0,0 +1,16 @@
+@function populate-breakpoints() {
```

Comment:
I'd probably put this file in the root of `scss/tools`, as it's more of a tool than a setting.

Reply:
Will do 🚯